src/main/kotlin/com/msg/gcms/global/webhook/util/impl/DiscordUtilImpl.kt

```diff
@@ -0,0 +1,66 @@
+package com.msg.gcms.global.webhook.util.impl
+
+import com.msg.gcms.domain.club.enums.ClubType
+import com.msg.gcms.global.webhook.util.DiscordUtil
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+import org.slf4j.LoggerFactory
+
+@Component
+class DiscordUtilImpl(
+    private val httpClient: OkHttpClient
+): DiscordUtil {
```

Comment:
```suggestion
) : DiscordUtil {
```
상속, 또는 구현할때 콜론의 앞 뒤에는 공백이 들어가는게 컨벤션으로 있어요!

Reply:
확인했습니다! 앞으론 더 신경써볼게요😅